Makefile

```diff
@@ -125,6 +125,7 @@ run-rootfs: bios/OVMF.fd bios/EFI
 # it gets triggered when we build any kind of image target
 images/%.yml: zedctr-workaround parse-pkgs.sh images/%.yml.in FORCE
 	DOCKER_ARCH_TAG="$(DOCKER_ARCH_TAG)" ./parse-pkgs.sh $@.in > $@
+	echo "Building version" `grep contents: $@`
```

Comment:
This will only work for the rootfs.yml, but won't work for things like installer.yml